service_management/azure/test/integration/vm/vm_create_test.rb

```diff
@@ -0,0 +1,300 @@
+#-------------------------------------------------------------------------
+# Copyright 2013 Microsoft Open Technologies, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#--------------------------------------------------------------------------
+require 'integration/test_helper'
+
+describe Azure::VirtualMachineManagementService do
+  subject { Azure::VirtualMachineManagementService.new }
+  let(:cloud_mgmt_service) { Azure::CloudServiceManagementService.new }
+  let(:vm_name) { 'testvm2321v' }
+  let(:linux_vm_image) { '03f55de797f546a1b29d1b8d66be687a__CoreCLR-x64-Beta5-Linux-PartsUnlimited-Demo-App-201504.29' }
```

Comment:
Should we reference a more basic VM image, in case this one goes away later? it shouldn't matter after recording is finished, but for cases we re-record.